e2e/tests/tap-tests.rs

```diff
@@ -0,0 +1,18 @@
+use crate::common::{exec_sql_script, run_postgres};
+use insta::assert_snapshot;
+use std::env;
+use test_generator::test_resources;
+use testcontainers::clients;
+
+mod common;
+
+#[test_resources("testdata/tap/*.sql")]
+fn tap_test(resource: &str) {
+    let docker = clients::Cli::default();
+    let node = run_postgres(&docker);
+    // Load the pgtap file. Note that this is manually generated using the `sql/pgtap.sql` target
+    // of the Makefile in the pgtap repository https://github.com/theory/pgtap.
+    exec_sql_script(&node, "testdata/pgtap-1.2.0.sql");
```

Comment:
Why create a new kind of test? I think it should be possible to use `\i` in those golden tests, that require it.